README.md

```diff
@@ -1,63 +1,87 @@
 # GlueSQL
+
 [![crates.io](https://img.shields.io/crates/v/gluesql.svg)](https://crates.io/crates/gluesql)
 [![docs.rs](https://docs.rs/gluesql/badge.svg)](https://docs.rs/gluesql)
 [![LICENSE](https://img.shields.io/crates/l/gluesql.svg)](https://github.com/gluesql/gluesql/blob/main/LICENSE)
 ![Rust](https://github.com/gluesql/gluesql/workflows/Rust/badge.svg)
 [![Chat](https://img.shields.io/discord/780298017940176946)](https://discord.gg/C6TDEgzDzY)
 
 ## SQL Database Engine as a Library
+
 GlueSQL is a SQL database library written in Rust. It provides a parser ([sqlparser-rs](https://github.com/ballista-compute/sqlparser-rs)), execution layer, and optional storage ([sled](https://github.com/spacejam/sled)) packaged into a single library.
-Developers can choose to use GlueSQL to build their own SQL database, or as an embedded SQL database using the default storage engine.  
+Developers can choose to use GlueSQL to build their own SQL database, or as an embedded SQL database using the default storage engine.
 
 ## Standalone Mode
+
 You can use GlueSQL as an embedded SQL database. GlueSQL provides [sled](https://github.com/spacejam/sled "sled") as a default storage engine.
 
 ### Installation
+
 In your `Cargo.toml`:
+
 ```toml
 [dependencies]
 gluesql = "0.7"
 ```
 
 ### Usage
+
 ```rust
-use gluesql::{parse, Glue, SledStorage};
+use gluesql::{parse, translate, Glue, SledStorage};
 
 fn main() {
     let storage = SledStorage::new("data.db").unwrap();
     let mut glue = Glue::new(storage);
 
     let sqls = "
+        DROP TABLE IF EXISTS Glue;
         CREATE TABLE Glue (id INTEGER);
         INSERT INTO Glue VALUES (100);
         INSERT INTO Glue VALUES (200);
         SELECT * FROM Glue WHERE id > 100;
-        DROP TABLE Glue;
     ";
-    
     for query in parse(sqls).unwrap() {
-        glue.execute(&query).unwrap();
+        let statement = translate(&query).unwrap();
+        let output = glue.execute_stmt(statement);
+        println!("{:?}", output);
```

Comment:
You can still use `glue.execute` though it can only run a single query for a sql string which includes multiple queries.
Code example here is for providing a simple and straightforward way to run the project almost without thinking.
Someone who wants more detail, they can figure it out `docs`, `tests` or `examples`.

In here, if you convert `sqls` into `Vec<_>`, you can keep using `execute`. like this
```rust
let sqls = vec![
    "DROP TABLE IF EXISTS Glue;",
    "CREATE TABLE Glue (id INTEGER);",
    ..
];

for sql in sqls {
    glue.execute(sql).unwrap();
}
```